### PR TITLE
158442962 header styles

### PR DESCRIPTION
**Pull request by rckopteri**
# Fixed
* Our tab component didn't handle mobile views very well.
   
# Changed
*Update header (h1-h4) styles

### CODE REVIEW

File: ote/src/cljs/ote/style/tabs.cljs [hunk 1]
```diff
@@ -5,10 +5,18 @@
 
 
 (def tab {:font-weight "bold"
-         :font-size "1em"
-         :display "inline"
-         :padding "0px 20px 10px 20px"
-         ::stylefy/mode {:hover {:border-bottom "4px solid #1976d2"}}})
-(def tab-selected (merge tab {:border-bottom "4px solid #1976d2" :color (color :blue700)}))
+          :font-size "1em"
```
Comment (K1ll3rF0x) [inline ote/src/cljs/ote/style/tabs.cljs]: We should use rem units here also.

File: ote/src/cljs/ote/ui/list_header.cljs [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-            [ote.ui.common :as common]))
+            [ote.ui.common :as common]
+            [ote.style.base :as style-base]))
 
-(defn header [title add-button operator-selection user-guide-button]
+(defn header [app title add-button operator-selection user-guide-button]
   [:div
    (when title
-     [:div.row [:h1 title]])
+     [:div.row (when (< (:width app) style-base/mobile-width-px) {:style {:padding-top "20px"}}) [:h1 title]])
```
Comment (K1ll3rF0x) [inline ote/src/cljs/ote/ui/list_header.cljs]: Any chance of adding this clause into a media query? I'd avoid these kind of conditional "surprises" in code if possible and define a media query that does the job in a stylefy style definition.

File: ote/src/cljs/ote/ui/tabs.cljs [hunk 3]
```diff
@@ -11,17 +11,20 @@
...
-  [:div.tab {:style {:padding-bottom "20px"}}
-   [:ul {:style {:list-style "none" :padding-bottom "12px"}}
+  [:div.tab {:style {:padding-bottom "40px"}}
     (doall
       (for [{:keys [label value]} tabs]
         ^{:key (str label)}
-        [:li {:style    {:display "inline"}
-              :on-click #(update-fn value)}
-         [:span (if (or (= value selected-tab) (and (nil? selected-tab) (= value (:value (first tabs)))))
+        [:div ;(merge
```
Comment (K1ll3rF0x) [inline ote/src/cljs/ote/ui/tabs.cljs]: Remove leftover comments ` ; ` and discards  `#_` in this section.

File: ote/src/cljs/ote/views/email_notification_settings.cljs [hunk 4]
```diff
@@ -39,7 +39,7 @@
 (defn email-notification-settings-form [e! state]
   (let [form-data (get-in state [:email-settings :user-notifications])]
     [:div
-     [:h1 (tr [:email-notification-settings-page :page-title])]
+     [ote.ui.list-header/header state (tr [:email-notification-settings-page :page-title])]
```
Comment (K1ll3rF0x) [inline ote/src/cljs/ote/views/email_notification_settings.cljs]: Require ote.ui.list-header :as list-header.

File: ote/src/cljs/ote/views/route.cljs [hunk 5]
```diff
@@ -29,7 +29,7 @@
 
 (defn form-container [e! app]
   [:div
-   [:h1 (tr [:common-texts :navigation-route])]
+   [ote.ui.list-header/header app (tr [:common-texts :navigation-route])]
```
Comment (K1ll3rF0x) [inline ote/src/cljs/ote/views/route.cljs]: Require ote.ui.list-header :as list-header.

File: ote/src/cljs/ote/views/user.cljs [hunk 6]
```diff
@@ -26,15 +26,15 @@
...
   (r/create-class
    {:component-will-unmount #(e! (lc/->CancelUserEdit false))
     :reagent-render
     (fn
       [e! {:keys [form-data email-taken username-taken password-incorrect?] :as user}]
 
-      [:div.user-edit.col-xs-12.col-md-6
-       [:h1 (tr [:common-texts :user-menu-profile])]
+      [:div.user-edit.col-xs-12.col-sm-8.col-md-8.col-lg-6
+       [ote.ui.list-header/header app (tr [:common-texts :user-menu-profile])]
```
Comment (K1ll3rF0x) [inline ote/src/cljs/ote/views/user.cljs]: Require ote.ui.list-header :as list-header.

File: ote/src/cljs/ote/ui/tabs.cljs (hunk 3 above)
Comment (rckopteri) [reply ote/src/cljs/ote/ui/tabs.cljs]: yep. my bad

File: ote/src/cljs/ote/ui/list_header.cljs (hunk 2 above)
Comment (rckopteri) [reply ote/src/cljs/ote/ui/list_header.cljs]: Didn't even think about using stylefy for this, but it worked quite nicely.

**Pull request merged**